src/components/CurrencyIcon.js

```diff
@@ -3,11 +3,10 @@
 import React, { PureComponent } from "react";
 import { View, StyleSheet } from "react-native";
 import { getCryptoCurrencyIcon } from "@ledgerhq/live-common/lib/reactNative";
-import type { CryptoCurrency } from "@ledgerhq/live-common/lib/types";
 import LText from "./LText";
 
 type Props = {
-  currency: CryptoCurrency,
+  currency: *,
```

Comment:
either we need to revert, or you can try to `currency: Currency` and handle more cases.
(in the future we'll need to implement it so it defaults to a rendering with the ticker anyway)

Reply:
I had it with Currency but there's no color there and it also breaks the icon, this was the cleanest even if not the most flow happy

Reply:
I think we can use `Currency` and put a FlowFixMe.

i'll discuss with flow folks to see if there is a solution for our problem, idk why Flow struggle solving this.

Reply:
There's no way I can get it to work with `Currency`, `CryptoCurrency` with flowfixme yes

Reply:
k we'll fix later